src/checkers/cwe_248.ml

```diff
@@ -0,0 +1,89 @@
+open Bap.Std
+open Core_kernel.Std
+
+let name = "CWE248"
+let version = "0.1"
+
+(* Print the findings to the log *)
+let print_uncatched_exception block_tid ~tid_map =
+   Log_utils.warn
+     "[%s] {%s} (Possibly Uncaught Exception) (Exception thrown at %s)."
+     name
+     version
+     (Address_translation.translate_tid_to_assembler_address_string block_tid tid_map)
+
+
+(* Returns the symbol name if the block contains a direct jump to a symbol *)
+let extract_direct_call_tid block =
+  let jmp_instructions = Term.enum jmp_t block in
+  Seq.fold jmp_instructions ~init:None ~f:(fun already_found instr ->
+    match already_found with
+      | Some(symb) -> Some(symb)
+      | None ->
+        match Jmp.kind instr with
+          | Goto _ | Ret _ | Int (_,_) -> None
+          | Call dst -> match Call.target dst with
+            | Direct tid ->
+              (* Tid.name tid is the name of the symbol.*)
+              Some(tid)
+            | _ -> None)
+
+(* Extract the Tid of a direct call, if the block contains a direct call. *)
+let extract_direct_call_symbol block =
+  match extract_direct_call_tid block with
+    | Some(tid) -> Some(Tid.name tid)
+    | None -> None
+
+(* check whether block contains a direct call to a symbol with name symbol_name *)
```

Comment:
we could refactor this function. Extract it as blk_contains_symbol and add it to utils/symbol_utils.ml